target/linux/ath79/dts/ar9341_engenius_ens202ext.dts

```diff
@@ -0,0 +1,150 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/dts-v1/;
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+#include "ar9341.dtsi"
+
+/ {
+	model = "Engenius ENS202EXT";
+	compatible = "engenius,ens202ext", "qca,ar9341";
+
+	aliases {
+		serial0 = &uart;
+		led-boot = &system;
+		led-failsafe = &system;
+		led-running = &system;
+		led-upgrade = &system;
+	};
+
+	keys {
+		compatible = "gpio-keys-polled";
+		poll-interval = <20>;
+
+		reset {
+			label = "Reset";
+			linux,code = <KEY_RESTART>;
+			gpios = <&gpio 1 GPIO_ACTIVE_LOW>;
+			debounce-interval = <60>;
+		};
+	};
+
+	leds {
+		compatible = "gpio-leds";
+
+		system: power {
+			label = "ens202ext:amber:power";
+			gpios = <&gpio 14 GPIO_ACTIVE_LOW>;
+			default-state = "on";
+		};
+
+		wlan0 {
+			label = "ens202ext:amber:wlan0";
+			gpios = <&gpio 18 GPIO_ACTIVE_LOW>;
+			linux,default-trigger = "phy0tpt";
+		};
+
+		rssilow {
+			label = "ens202ext:red:rssi1";
+			gpios = <&gpio 16 GPIO_ACTIVE_LOW>;
+		};
+
+		rssimedium {
+			label = "ens202ext:amber:rssi2";
+			gpios = <&gpio 17 GPIO_ACTIVE_LOW>;
+		};
+
+		rssihigh {
+			label = "ens202ext:green:rssi3";
+			gpios = <&gpio 0 GPIO_ACTIVE_LOW>;
+		};
+	};
+};
+
+&ref {
+	clock-frequency = <25000000>;
+};
+
+&gpio {
+	status = "okay";
+};
+
+&spi {
+	status = "okay";
+
+	num-cs = <1>;
+	flash@0 {
+		compatible = "jedec,spi-nor";
+		reg = <0>;
+		spi-max-frequency = <25000000>;
+
+		partitions {
+			compatible = "fixed-partitions";
+			#address-cells = <1>;
+			#size-cells = <1>;
+
+			uboot: partition@0 {
```

Comment:
do you need this DT label?

Reply:
I don't know actually...I guess not?

I used ar9341_tplink_tl-wr841-v8.dts as a base
(before it was split into ar9341_tplink.dtsi)

Reply:
You don't use it anywhere if I haven't overlooked it (&uboot), so you can just drop it (in contrast to art, which you need). For the old TP-Link devices, the MAC address is stored in the uboot partition, and there it is needed there.